COPYRIGHT

```diff
@@ -0,0 +1,7 @@
+threshold_crypto is copyright 2018, POA Network Developers.
```

Comment:
Do you think "POA Network" could also be OK? In addition, POA Network (Ltd) is a legal entity. Although I don't think we should write "POA Network Ltd."

Reply:
Yeah, we can put whatever there. @igorbarinov, what should we put?

Reply:
Why not use the company name? e.g. Facebook MIT license uses Facebook Inc. https://github.com/facebook/flow/blob/master/LICENSE as a grantor of the license. If it's ok let's use `POA Networks Ltd.`